src/pages/static/MarkdownReader.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { string } from 'prop-types';
+import remark from 'remark';
+import remark2react from 'remark-react';
+
+class MarkdownReader extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      text: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch(this.props.content)
+      .then((res) => res.text())
+      .then((text) => this.setState({ text }));
+  }
+
+  render() {
+    return (
+      <div>
+        {
+          remark()
+            .use(remark2react)
+            .processSync(this.state.text).contents
+        }
+      </div>
+    );
+  }
+}
+
+MarkdownReader.propTypes = {
+  content: string, // a URL to a .md file
+};
+
+export { MarkdownReader };
```

Comment:
just for consistency with the rest of the codebase, we are exporting using default:

```
export default MarkdownReader;
```

Reply:
That wasn't me! 😅 It was like that before I got here.

Happy to make the change though, as it's my preferred style too. This necessitated a change at the other end, where these components are imported in `src/pages/home/components/LegalModal.js`

Reply:
ha.... didn't noticed it was like that. Anyway, let's update it to be aligned with the rest of the code.